src/app/clusters/ota-requestor/OTARequestor.cpp

```diff
@@ -298,11 +301,6 @@ void OTARequestor::CancelImageUpdate()
 {
     mBdxDownloader->EndDownload(CHIP_ERROR_CONNECTION_ABORTED);
 
-    // All Provider info should be invalidated
-    mProviderNodeId      = kUndefinedNodeId;
```

Comment:
@selissia I removed this since we kind of always want `mProviderLocation` to contain something ready for the next query. We can always use the `mCurrentUpdateState` to determine if we should look at that value or not

Reply:
`mProviderLocation` would be empty after the initialization until the Provider info has been passed to the Requestor somehow -- so having it empty is a valid state. 
In general, I think it's always better to have empty info rather than stale info.

Reply:
Yes, the empty case is valid. But I also don't see it as stale info. When an update has ended (whether success or cancelled), we revert back to the default OTA provider. That is valid to be used for the next query.